cmd/create.go

```diff
@@ -170,18 +172,35 @@ var createCmd = &cobra.Command{
 		}
 
 		if flagAdvanced {
-			isInteractive = true
-			step := steps.Steps["advanced"]
-			tprogram = tea.NewProgram((multiSelect.InitialModelMultiSelect(step.Options, options.Advanced, step.Headers, project)))
-			if _, err := tprogram.Run(); err != nil {
-				cobra.CheckErr(textinput.CreateErrorInputModel(err).Err())
-			}
-			project.ExitCLI(tprogram)
-			for key, opt := range options.Advanced.Choices {
-				project.AdvancedOptions[key] = opt
-			}
-			if err != nil {
-				log.Fatal("failed to set the htmx option", err)
+
+			featureFlags := cmd.Flag("feature").Value.String()
+			hasFeatureFlags := featureFlags != ""
+
+			if hasFeatureFlags {
```

Comment:
```suggestion
			featureFlags := cmd.Flag("feature").Value.String()

			if featureFlags != "" {
```

We can remove the extra variable and just do the check right on `featureFlags`

Reply:
Commited!